1D CSR Model (#627)
BLAST-ImpactX/impactx: docs/source/usage/parameters.rst

Coherent Synchrotron Radiation (CSR)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> CSR effects are included in the simulation for bend lattice elements such as Sbend and CFbend. These effects are critical in accurately modeling the wakefields generated due to the interaction of particles with the radiated field during bending in synchrotron radiation.

Reviewer: ```suggestion
CSR effects are included in the simulation for bend lattice elements such as Sbend and CFbend. These effects are critical in accurately modeling the wakefields generated due to the interaction of particles with the synchrotron radiation field generated by the beam during bending.  Currently, this is the 1D ultrarelativistic steady-state wakefield model (eq 19 of
E. L. Saldin et al, NIMA 398, p. 373-394 (1997), [DOI:10.1016/S0168-9002(97)00822-X]).
```

* ``algo.csr`` (``boolean``, optional, default: ``false``)
    Whether to calculate CSR effects.

>     CSR calculations involve several steps, including charge deposition, mean transverse position calculation, wakefield generation, and convolution, all of which are handled within the CSR bending process.

Reviewer: ```suggestion
    CSR calculations involve several steps, including charge deposition, wakefield generation, and convolution, all of which are handled within the CSR bending process.
```

Reviewer: The mean transverse position calculation is not used for CSR calculations.

> * ``algo.csr`` (boolean, optional, default: false)
>     Whether to calculate CSR effects.
>     CSR calculations involve several steps, including charge deposition, mean transverse position calculation, wakefield generation, and convolution, all of which are handled within the CSR bending process.
> 
> * ``algo.csr_bins`` (integer, optional, default: 150)

Reviewer 2: Verbatim formatting as in other parameters:
```suggestion
* ``algo.csr`` (``boolean``, optional, default: ``false``)
    Whether to calculate CSR effects.
    CSR calculations involve several steps, including charge deposition, mean transverse position calculation, wakefield generation, and convolution, all of which are handled within the CSR bending process.

* ``algo.csr_bins`` (``integer`, optional, default: ``150``)
```

> Whether to calculate CSR effects.
> CSR calculations involve several steps, including charge deposition, mean transverse position calculation, wakefield generation, and convolution, all of which are handled within the CSR bending process.
> 
> * ``algo.csr_bins`` (integer, optional, default: 150)
> The number of bins used for the CSR calculations along the longitudinal direction. Increasing the number of bins can lead to more accurate wakefield resolution at the cost of higher computational expense.
> 
> .. note::
> 
> CSR effects are only calculated for lattice elements that include bending, such as Sbend and CFbend.

Reviewer 2: These spaces are needed so Sphinx/rst formats this correctly (bullets & blocks):
```suggestion
    Whether to calculate CSR effects.
    CSR calculations involve several steps, including charge deposition, mean transverse position calculation, wakefield generation, and convolution, all of which are handled within the CSR bending process.

* ``algo.csr_bins`` (integer, optional, default: 150)
    The number of bins used for the CSR calculations along the longitudinal direction. Increasing the number of bins can lead to more accurate wakefield resolution at the cost of higher computational expense.

.. note::

   CSR effects are only calculated for lattice elements that include bending, such as Sbend and CFbend.
```
